### PR TITLE
Removing references to migrating from DB v10

### PR DESCRIPTION
**Pull request by 0rnela**
# Description

Uyuni docs still mention DB migration scripts from v10. We only need 12 -> 13.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/upgrade/pages/server-major-upgrade-uyuni.adoc [hunk 1]
```diff
@@ -31,37 +37,16 @@ zypper ref
...
-+
-----
-/usr/lib/susemanager/bin/pg-migrate-10-to-13.sh
 ----
 +
 For more information about PostgreSQL database migration, see xref:db-migration-13.adoc[].
 . After the migration is complete, manually reboot the {productname} Server:
 +
 ----
-shutdown -r now
```
Comment (juliogonzalez) [inline modules/upgrade/pages/server-major-upgrade-uyuni.adoc:58]: Why are we removing this, and the IMPORTANT note?

**Review by juliogonzalez — COMMENTED** [review]
LGTM, but I have one question about some things that are removed

**Comment by 0rnela** [comment]
Attn @juliogonzalez :

[suse_manager_upgrade_guide.pdf](https://github.com/uyuni-project/uyuni-docs/files/6686387/suse_manager_upgrade_guide.pdf)
[uyuni_upgrade_guide.pdf](https://github.com/uyuni-project/uyuni-docs/files/6686388/uyuni_upgrade_guide.pdf)

**Comment by juliogonzalez** [comment]
Seems we have some problems for Uyuni.

image

And the following links:

image

Maybe you need to adjust the indexes, so 10-12 is not linked anymore (for Uyuni), and the other says 12-13 only.

The SUSE Manager PDF looks good.

**Comment by 0rnela** [comment]
[suse_manager_upgrade_guide.pdf](https://github.com/uyuni-project/uyuni-docs/files/6686720/suse_manager_upgrade_guide.pdf)
[uyuni_upgrade_guide.pdf](https://github.com/uyuni-project/uyuni-docs/files/6686723/uyuni_upgrade_guide.pdf)

**Comment by 0rnela** [comment]
Two upgrade guides, side by side.

SUMA one is missing Troubleshooting guide in the navigation bar.

[suse_manager_upgrade_guide.pdf](https://github.com/uyuni-project/uyuni-docs/files/6687564/suse_manager_upgrade_guide.pdf)
[uyuni_upgrade_guide.pdf](https://github.com/uyuni-project/uyuni-docs/files/6687565/uyuni_upgrade_guide.pdf)

**Pull request merged**